docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/xcluster-replication-setup.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Set up xCluster replication for a universe
+headerTitle: Set up xCluster replication
+linkTitle: Setup
+description: Setting up xCluster replication for a universe
+headContent: Start replication from your source to your target
+menu:
+  preview_yugabyte-platform:
+    parent: xcluster-replication
+    identifier: xcluster-replication-setup
+    weight: 10
+type: docs
+---
+
+## Prerequisites
+
+Create two universes, the source universe which will serve reads and writes, and the target.
+
+Ensure the universes have the following characteristics:
+
+- Both universes are running the same version of YugabyteDB (v2.18.0.0 or later).
+- Both universes have the same [encryption in transit](../../../security/enable-encryption-in-transit/) settings. Encryption in transit is recommended, and you should create the source and target universes with TLS enabled.
```

Comment:
```suggestion
- Both universes have [node to node encryption](../../../security/enable-encryption-in-transit/) enabled (recommended) or both have this setting disabled.
```

The above statement seems like we require TLS for xcluster and that we require the same TLS settings (CA etc).